environment.yml

```diff
@@ -2,13 +2,10 @@ channels:
   - conda-forge
 dependencies:
   - cmake >=3.20
-  - cudatoolkit 11.8.*
```

Comment:
Don't remove dependencies. The file should work for the build from source too (https://github.com/openmm/NNPOps#build-from-source)

Reply:
I'm in the middle of playing with this, I got AWS GPU stuff working :sunglasses: My plan is once it is all working, move the env file for CI to either a different folder or name, so that the installation instructions are still correct, but it will make changing the build matrix easier using `extra-specs` instead of `sed` to modify the environment file on the fly

Reply:
I would prefer to use a single environment file and modify it as needed. This ensures consistency.

Reply:
I believe we should switch to mamba-org if it is faster.